ament_copyright/ament_copyright/main.py

```diff
@@ -217,8 +217,10 @@ def main(argv=sys.argv[1:]):
 
 
 def add_missing_header(file_descriptors, name, license_, verbose):
-    copyright_ = 'Copyright %d %s' % (int(time.strftime('%Y')) - 1 + 1, name)
-    header = license_.file_header.format(**{'copyright': copyright_})
+    copyright_ = 'Copyright %d, %s' % (int(time.strftime('%Y')) - 1 + 1, name)
```

Comment:
The majority of existing code doesn't seem to have a comma between the year and the name. Why should that be changed in the template?

Reply:
I was thinking in some new licenses notices I have seen with the comma, I will remove it to avoid changing the standard.